bedrock-entry.sh

```diff
@@ -190,9 +190,18 @@ set-property --file server.properties --bulk /etc/bds-property-definitions.json
 
 export LD_LIBRARY_PATH=.
 
+mcServerRunnerArgs=()
+if isTrue "${ENABLE_SSH}"; then
+  mcServerRunnerArgs+=(--remote-console)
+  if ! [[ -v RCON_PASSWORD ]]; then
+    RCON_PASSWORD=$(openssl rand -hex 12)
```

Comment:
I'm wondering: why even enable ssh if they didn't supply an rcon password and a random one is generated without "persistence"? On the Java image I write out the random password to an internal file so that it can be used (barely securly) by rcon-cli running within the container and the itzg/mc-backup image. 

Perhaps logging it would be a decent compromise. If a bad actor has the ability to dump container logs then they can already do lots of other bad things.

Reply:
What I was aiming for was behavior parity with the Java container. If you enable SSH or RCON there without setting the password, you can “connect” but you always get the password rejected. This felt intuitive to me as the first thing that came to mind was: “Why is my password wrong?” not “Why is the server not reachable?” It’s minor for sure, but I think being able to get hints of what step you missed even if you never look at the logs has value. 

Having both containers behave the same way also makes it a bit easier for someone to manage a mix of servers for whatever reason due to less cognitive load having to think about the differences. 

But that’s just my thinking. I’ll defer to what you want here.

Reply:
But that's not actually the intended behavior for the Java container. Again, it writes out the per-run generated password so that rcon-cli and mc-backup can use it:

https://github.com/itzg/docker-minecraft-server/blob/aec644dc3f81d8847d86cbc73ea6678e8ca54dcb/scripts/start-configuration#L78

Now, your tooling probably simply rejects an unset/generated password, but that's only because you not leveraging the existing mechanism 😀

Reply:
Ah, you set $HOME to the data folder, so the files are accessible to anything that has access to the data volume. That's what I was missing.

I actually think the right call is to do the same here then. Maybe calling the files rcon-cli.yaml and rcon-cli.env would be a little weird though. 

> Now, your tooling probably simply rejects an unset/generated password, but that's only because you not leveraging the existing mechanism

I think I'm being misunderstood here. I'm thinking of it from a less aware user working their way through things that may not be aware of due to the sheer complexity/functionality available. Yes, my tooling will not work if the password isn't set there, but say a user configures _a_ password on the backup container, but forgets to set it on the mc container (something I actually did). It's less intuitive for even someone like me to understand that a missing/misconfigured password caused a "port closed" error, and so less savvy users will be less able to make the connection.

Reply:
> I actually think the right call is to do the same here then. Maybe calling the files rcon-cli.yaml and rcon-cli.env would be a little weird though. 

Agreed, doesn't need to be written to the same file names. Eve more so, those were dictated by cobra library used in rcon-cli. Something generic like `remote-console.env` would be my suggestion. 

> I think I'm being misunderstood here. I'm thinking of it from a less aware user working their way through things that may not be aware of due to the sheer complexity/functionality available. Yes, my tooling will not work if the password isn't set there, but say a user configures _a_ password on the backup container, but forgets to set it on the mc container (something I actually did). It's less intuitive for even someone like me to understand that a missing/misconfigured password caused a "port closed" error, and so less savvy users will be less able to make the connection.  

We're basically getting at the same thing, but my goal is zero config should simply work. In the case of my backup image, don't configure rcon password explicitly and everything works. Sure, if they go and configure a password in the backup container but not the server container  that won't work, but that was a conscious action the user took. (However, when asking for support human nature will dictate they forgot or copy pasted an example and are baffled why it doesn't work 😀)

Reply:
I've cleaned up the file name, and it's working well with my container, so I think we've resolved this thread. 

Now I just need to finish cleaning up the new examples to match the work here and push the changes for my container as well.